src/jit/lsraxarch.cpp

```diff
@@ -1978,13 +1978,15 @@ void LinearScan::TreeNodeInfoInitIntrinsic(GenTree* tree)
             }
             break;
 
-#ifdef _TARGET_X86_
+#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
```

Comment:
This isn't needed here; this file is only for these two targets.

Reply:
Fixed.